roles/validations/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# We can execute all defined validations when cifmw_validation_run_all is defined.
+# Else, we will skip this and run only the explicitly defined validations from
+# cifmw_validations_list
+- name: Run all validations if required
+  when: cifmw_validations_run_all | bool
+  block:
+    - name: Find all validations
+      register: found_validations
+      ansible.builtin.find:
+        path: "{{ role_path }}/tasks"
```

Comment:
we may play a bit here. Not sure if it would work, but:
`path: "{{ cifmw_validation_default_path }}"`

with, in the `defaults/main.yml`, `cifmw_validation_default_path: "{{ role_path }}/tasks"`

This MAY allow to set an external path, for instance if someone has another (downstream-only?) set of validations to run. Might be good to test if the `include_tasks` would work with that, but since it accepts a path, I don't see why it wouldn't.

Thoughts?

Reply:
Sounds good. I implemented this in the latest iteration.